dev/io.openliberty.netty.internal/bnd.bnd

```diff
@@ -0,0 +1,33 @@
+#*******************************************************************************
+# Copyright (c) 2020 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+#*******************************************************************************
+
+-include= ~../cnf/resources/bnd/bundle.props
+bVersion=1.0
+nettyVersion=4.1.51.Final
+
+Bundle-Name: Netty as an OSGi bundle
+Bundle-SymbolicName: io.openliberty.netty.internal
+Bundle-Description: Netty as an OSGi bundle for OpenLiberty, version ${bVersion}
+
+-includeresource: \
+  @${repo;io.netty:netty-buffer;${nettyVersion};EXACT}, \
+  @${repo;io.netty:netty-codec;${nettyVersion};EXACT}, \
+  @${repo;io.netty:netty-codec-http;${nettyVersion};EXACT}, \
+  @${repo;io.netty:netty-codec-http2;${nettyVersion};EXACT}, \
+  @${repo;io.netty:netty-codec-socks;${nettyVersion};EXACT}, \
+  @${repo;io.netty:netty-common;${nettyVersion};EXACT}, \
+  @${repo;io.netty:netty-handler;${nettyVersion};EXACT}, \
+  @${repo;io.netty:netty-handler-proxy;${nettyVersion};EXACT}, \
+  @${repo;io.netty:netty-resolver;${nettyVersion};EXACT}, \
+  @${repo;io.netty:netty-transport;${nettyVersion};EXACT}
+
+Export-Package: \
+  io.netty.*;version=${nettyVersion} -split-package:=merge-first
```

Comment:
What does `-split-package:=merge-first` do? Split packages are to be avoided if possible.

Reply:
This is needed until the netty packages are removed from grpc.
This is just a draft PR at the moment.

Reply:
You need a semicolon before `-split-packages`

```properties
Export-Package: \
  io.netty.*;version=${nettyVersion};-split-package:=merge-first
```

Reply:
Added